CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -577,7 +577,7 @@ public function parseActionSchedule($values) {
 
     $params['is_active'] = CRM_Utils_Array::value('is_active', $values, 0);
 
-    if (CRM_Utils_Array::value('is_repeat', $values) == 0) {
+    if (empty($values['is_repeat'])) {
```

Comment:
ok - I guess is_repeat is a boolean - although that sort of thing always feels like a big assumption with our code

Reply:
Well since this statement uses loose `==` and not strict `===`, in this case the two statements are equivalent.